app/javascript/store.js

```diff
@@ -0,0 +1,21 @@
+import { configureStore, createSlice } from '@reduxjs/toolkit';
+
+const greetingSlice = createSlice({
+  name: 'greeting',
+  initialState: {
+    randomGreeting: '',
+  },
+  reducers: {
+    setRandomGreeting: (state, action) => {
+      state.randomGreeting = action.payload;
+    },
+  },
+});
+
+export const { setRandomGreeting } = greetingSlice.actions;
```

Comment:
[OPTIONAL] Kindly remove this code because you already created a greetingSlice in side the greetingSlice.js file so import that file and use it in that file :slightly_smiling_face: